### PR TITLE
Fix reset

### PR DESCRIPTION
**Pull request by benrudolph**
@czue this fixes the checkpoint reset function

cc: @esoergel

### CODE REVIEW

File: pillowtop/tests/test_checkpoints.py [hunk 1]
```diff
@@ -23,14 +28,6 @@ def test_get_or_create_empty(self):
         self.assertEqual('0', checkpoint['seq'])
         self.assertTrue(bool(checkpoint['timestamp']))
 
-
-class PillowCheckpointTest(SimpleTestCase):
```
Comment (benrudolph) [inline pillowtop/tests/test_checkpoints.py]: This is unrelated to the fix, but when I was searching for a place to add the test I found that two classes were named the same and thus the first one wasn't being run

**Comment by esoergel** [comment]
Looks good to me.  Is there a corresponding FB ticket for this?

**Comment by benrudolph** [comment]
no ticket, ran into this when trying to get migrations to run on travis

File: pillowtop/tests/test_checkpoints.py (hunk 1 above)
Comment (czue) [inline pillowtop/tests/test_checkpoints.py]: yikes, i have no idea how that happened. thanks.

**Comment by czue** [comment]
thanks for fixing and sorry about that!

**Pull request merged**